app/services/stock/routes/equityRoutes.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const router = express.Router();
+const equityController = require('../controllers/equityController.js');
+
+router.get('/:ticker', equityController.equity);
```

Comment:
Merge this file with stockRoutes.js in #38 to allow everything to use the `/equity` root. 

Currently have:
- `/equity/intraday/:ticker`
- `/equity/historical/:ticker`

Maybe use `/equity/statistics/:ticker` or something similar for `equityController.equity`?

Reply:
fixed